template/apache2.conf

```diff
@@ -30,12 +30,12 @@ Listen 80
 	Require all denied
 </Directory>
 
-<Directory /app>
+<Directory ${VOLUME_PATH}>
```

Comment:
i wanted to double check, but I am assuming this is Ok? `DocumentRoot` is now the same as directory path, while they were different before.